### PR TITLE
[libxlsxwriter] upgrade to 0.9.4

### LINKED ISSUE #9772: [libxlsxwriter] update to 0.9.4
**Library name**: libxlsxwriter

**New version number**: 0.9.4

**Other information that may be useful (release notes, etc...)**: https://libxlsxwriter.github.io/changes.html

### PR DESCRIPTION
**Pull request by jmcnamara**
Upgrade to the latest version of libxlsxwriter.

Closes #9772

### CODE REVIEW

File: ports/libxlsxwriter/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: libxlsxwriter
-Version: 0.8.9-1
+Version: 0.9.4-1
```
Comment (NancyLi1013) [inline ports/libxlsxwriter/CONTROL]: The version should be 0.9.4. 
Could you please update it?

File: ports/libxlsxwriter/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 Description: Libxlsxwriter is a C library that can be used to write text, numbers, formulas and hyperlinks to multiple worksheets in an Excel 2007+ XLSX file.
 Build-Depends: zlib
```
Comment (NancyLi1013) [inline ports/libxlsxwriter/CONTROL]: Could you please add homepage here?
`Homepage: https://github.com/jmcnamara/libxlsxwriter`

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/libxlsxwriter/CONTROL (hunk 2 above)
Comment (jmcnamara) [reply ports/libxlsxwriter/CONTROL]: Done. Thanks for the review.

File: ports/libxlsxwriter/CONTROL (hunk 1 above)
Comment (jmcnamara) [reply ports/libxlsxwriter/CONTROL]: Done

**Review by NancyLi1013 — APPROVED** [review]

**Comment by vicroms** [comment]
Thanks for the PR!

**Comment by jmcnamara** [comment]
Thanks in return for the reviews and the nice packager.

**Pull request merged**